### PR TITLE
Added coverage report

### PR DESCRIPTION
**Pull request by oed**
This PR adds a coverage report using solidity-coverage. It also adds some additional tests.
The coverage report will be availiable on https://uport-project.github.io/uport-identity/coverage once this branch is in master. They are also availiable in the `docs/coverage` folder

### CODE REVIEW

**Comment by oed** [comment]
Some things I found that might be worth updating in the `(Meta)IdentityManager` contracts while updating the tests:

* `migrationInitiated` & `migrationNewAddress` should maybe be public?
* Check in `finalizeMigration` should throw, otherwise there is no way of knowing if the call failed
* Add `isOlderOwner` constant function?

File: docs/coverage/lcov-report/sorter.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
+var addSorting = (function () {
```
Comment (naterush) [inline docs/coverage/lcov-report/sorter.js]: Hm. Ok, just to clarify and make sure I understand correctly, the .html files above are essentially ways of displaying the coverage of the contracts. Is this file for helping with that task also?

File: docs/coverage/sorter.js [hunk 2]
```diff
@@ -0,0 +1,158 @@
+var addSorting = (function () {
```
Comment (naterush) [inline docs/coverage/sorter.js]: Hmm - is this a file repeat? Not sure I totally follow what is happening here :)

Comment (oed) [inline docs/coverage/sorter.js]: all files in the `coverage` folder are auto generated :)

**Pull request merged**